postgres/datadog_checks/postgres/config.py

```diff
@@ -2,322 +2,467 @@
 # All rights reserved
 # Licensed under Simplified BSD License (see LICENSE)
 # https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PARAMKEYWORDS
-from typing import Optional
+from __future__ import annotations
 
-from datadog_checks.base import AgentCheck, ConfigurationError, is_affirmative
+import copy
+from typing import TYPE_CHECKING, Optional, Tuple
+
+from datadog_checks.postgres.config_models import InstanceConfig
+from datadog_checks.postgres.config_models.defaults import (
+    instance_dbm,
+    instance_disable_generic_tags,
+    instance_port,
+    instance_propagate_agent_tags,
+)
```

Comment:
These can be removed and their usage below can use the `defaults.instance_*`. No need to explicitly import again